terraform/runner/README.md

```diff
@@ -0,0 +1,100 @@
+# gitlab-runner-cloudgov terraform
+Terraform for running GitLab CI/CD jobs on cloud.gov or another CloudFoundry based PaaS.
+
+* [Deploying](#deploying)
+* [Troubleshooting](#troubleshooting)
+* [Design Decisions](#design-decisions)
+
+## Deploying
+
+1. Log in to cloud.gov
+    ```
+    cf login -a api.fr.cloud.gov --sso
+    ```
+
+2. Target the org and space for deployment
+    ```
+    cf target -o ORGNAME -s SPACENAME
+    ```
+    For example:
+    ```
+    cf target -o sandbox-gsa -s bret.mogilefsky
```

Comment:
nitpick (non-blocking): I know this is pulled from some other READMEs. Would be good to remove Bret's specific space name.

Reply:
Changed to sally.sassalot